app/uk/gov/hmrc/uknwauthcheckerapi/config/AppConfig.scala

```diff
@@ -19,12 +19,20 @@ package uk.gov.hmrc.uknwauthcheckerapi.config
 import javax.inject.{Inject, Singleton}
 
 import play.api.Configuration
+import uk.gov.hmrc.http.StringContextOps
 
 @Singleton
 class AppConfig @Inject() (config: Configuration) {
 
-  val authType: String = config.get[String]("authType")
+  private lazy val rootServices = "microservice.services"
+  private lazy val defaultProtocol: String =
```

Comment:
Dan, defaultProtocol val can be removed as we have getConfString serves the same

Reply:
defaultProtocol is a required parameter in getConfString. If we remove it we must supply a default.

Reply:
I mean we can call getConfString(s"$serviceName.protocol", "http") as I see duplicated code from 28 to 31 and 46 to 49

Reply:
Good shout. Done